backend/tests/test_project.py

```diff
@@ -1,7 +1,7 @@
 import json
 import unittest
 import uuid
-from flask import current_app
+from flask import current_app, request
```

Comment:
@peterghrong I think this is an unused import?

Reply:
Yes, missed it mb